_scalingo_api/log_drains.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Log drains
+layout: default
+---
+
+# Log drains
+
+Log from an application can be redirect to a management service.
+You can add multiple log drains from a single application.
+You can also remove or list log drains by application.
+
+You will find how to enable this service from the CLI on the <a href="https://doc.scalingo.com/platform/app/log-drain">public documentation</a>
+
+--- row ---
+
+**Supported protocol and logs management services with their parameter**
+
+{:.table}
+| Logs management service   | type        | token | host | port | drain_region | url | will format urls:                                    |
```

Comment:
I think the "will format urls" column is useless. Customers don't care, it's kinda just an internal representation.